src/frontend/src/pages/ProjectDetailPage/ProjectViewContainer/WorkPackageSummary.tsx

```diff
@@ -44,8 +54,18 @@ const WorkPackageSummary: React.FC<WorkPackageSummaryProps> = ({ workPackage })
     </ul>
   );
   const numMoreDeliverables = workPackage.deliverables.length - 3;
-
-  const theme = useTheme();
+  const dependencyList = (
```

Comment:
you can call this DependencyList = () => (react stuff), and then call it like `<DependencyList/>` below, it's a cool thing with React, just makes the code a lil cleaner